### ISSUE TITLE
I would like HCF version of this plugin

### ISSUE THREAD

**Issue by nscwalk** [drtshock/Factions#243]
I originally donated to this plugin because I was looking forward to this HCF feature in this plugin. But I can not figure how to get DTR added to the plugin. like it is only has power and nothing in the config files to activate DTR. So if there is like a separate factions plugin for hcf I would like it to be available because that originally what I wanted.

**Comment by drtshock** [comment]
Did you actually read and try configuring things?

**Comment by nscwalk** [comment]
YES AND I CAN NOT ACTIVATE DTR

**Comment by cameronb23** [comment]
@drtshock I don't believe he has the right version of the plugin....mine works perfectly fine :+1:

**Comment by nscwalk** [comment]
-removed stupid long paste of config-

**Comment by drtshock** [comment]
so 1. don't paste your config.. 2. maybe if you read what you pasted ._. as well as the conf.json

**Comment by nscwalk** [comment]
The Conf.Json I have does not have the HCF Features at the bottom That is my problem My factions is probaly outdated or something is wrong because when I downloaded the latest version of FactionsUUID or whatever did not come with hcf: Raidable:true/false DTR:True/False. And I have completely read what is in the conf.json.... there is something wrong with my factions.  and to prove that I have read the whole conf. its bassically what the default relationship between all factions is (enemy neutral ally) wheather people can create portals. Cost for warps. The f show list to show what will be up on here (I would like to show DTR Here but dont know how to add that) the scoreboard thing on the side of the screen showing the /f who/show prefixes and colors of peoples faction names. dely for f home / f warp. then we get into the part I didnt have by default that I had to paste into my conf.json. how many allies/enemies you want and if you want it enabled or disabled. weather you want DTR And Raidability on. I do I set this to true still nothing shows up.allow overclaiming of somones base and how long somones power should be frozen until regen bufferzone. how far people must claim from each other deterimained by chunks. I have read the whole conf.json but there is something wrong with my version of factions that is not letting me enable the hcf features.
